rd-net/RdFramework/Base/ISingleContextHandler.cs

```diff
@@ -1,12 +1,12 @@
+using System;
 using JetBrains.Serialization;
 
 namespace JetBrains.Rd.Base
 {
   internal interface ISingleContextHandler
   {
-    void ReadValueAndPush(SerializationCtx context, UnsafeReader reader);
```

Comment:
_(Minor)_ We could win a couple of boxing/indirect calls here by exposing `UpdateValueCookie` through this interface and making it a `struct`, but I'm not sure it's worth it.

Reply:
Decided to let it be for now. I'll think more about improvements in this area later.